javascript/javascript2/week3/script.js

```diff
@@ -0,0 +1,128 @@
+// 1. Logout text
+let logOut = () => {
+  console.log('Script loaded');
+};
+setTimeout(logOut, 2500);
+
+// 2. Delay function
+
+function setTime(n, callback, time = 0) {
+  setTimeout(() => {
```

Comment:
I would suggest to define time as a variable inside the code if you are not going to pass in to the function!